src/module/api/role/search/index.ts

```diff
@@ -48,6 +48,7 @@ export async function Search(searchRole: IAPISearchRole): Promise<IRole[]> {
             role: roleKind,
             subjectId,
             ventureId,
+            timelineId: searchRole.timelineId,
```

Comment:
Ideally this would come from the API response, but I think it's safe to assume that the timeline that is passed in to the request will be the timeline of the resources that are returned. This allows us to match up roles with timelines correctly (we originally only had the role ID, the subject (i.e. user) ID, and venture ID).